packages/client/lib/rpc/modules/debug.ts

```diff
@@ -11,9 +12,49 @@ export interface tracerOpts {
   disableStorage?: boolean
   enableMemory?: boolean
   enableReturnData?: boolean
+  tracer?: string
+  timeout?: string
+  tracerConfig?: any
 }
+
 /**
- * web3_* RPC module
+ * Validate tracer opts to ensure only supports opts are provided
+ * @param opts a dictionary of {@link tracerOpts}
+ * @returns a dictionary of valid {@link tracerOpts}
+ * @throws if invalid tracer options are provided
+ */
+const validateTracerConfig = (opts: tracerOpts): tracerOpts => {
+  if (opts.tracerConfig !== undefined) {
+    throw {
+      code: INVALID_PARAMS,
+      message: 'custom tracers and tracer configurations are not implemented',
+    }
+  }
+  if (opts.tracer !== undefined) {
+    throw {
+      code: INVALID_PARAMS,
+      message: 'custom tracers not implemented',
+    }
+  }
+  if (opts.timeout !== undefined) {
+    throw {
+      code: INVALID_PARAMS,
+      message: 'custom tracer timeouts not implemented',
+    }
+  }
+  if (opts.disableStorage === false) {
+    throw {
+      code: INVALID_PARAMS,
+      message: 'storage retrieval not implemented',
+    }
+  }
+  return {
+    ...{ disableStack: false, disableStorage: true, enableMemory: true, enableReturnData: false },
```

Comment:
Why are two of these values defaulted to different values than in geth? See:
![image](https://user-images.githubusercontent.com/18757482/206738896-82f9deb5-964f-48bf-8cfd-bb375ed4f92b.png)

Reply:
Mainly because I'm not sure how to implement them. I can think on it some but they're tricky, especially `returnData`

Reply:
Ah yeah I didn't mean that we should add support for them, but just questioning the fact that we default to other values. Either we:
- Default to the same values as geth, which means that we won't handle the default case where all booleans are false
- Default to different values as geth, which means that our API will differ from the standard. The benefit being that we support the default case (since we've adjusted it for something we support)
I would favor the first option. wdyt?

Reply:
I switched things around so the defaults should be supported now.  Note, I haven't implemented the `enableReturnData === true` option as that requires keeping track of call frames and returning the last call's returnValue and beyond the scope of what I want to implement for now.